src/sst/core/configGraphOutput.cc

```diff
@@ -17,15 +17,12 @@
 #include "sst/core/simulation_impl.h"
 #include "sst/core/util/filesystem.h"
 
-namespace SST {
-namespace Core {
+namespace SST::Core {
 
 ConfigGraphOutput::ConfigGraphOutput(const char* path)
 {
     SST::Util::Filesystem filesystem = Simulation_impl::filesystem;
     outputFile                       = filesystem.fopen(path, "wt");
 }
 
-
-} // namespace Core
-} // namespace SST
+} // namespace SST::Core
```

Comment:
There were still a few places where nested namespace names were not used. `clang-tidy` was able to fix these automatically, although it sometimes left the old namespace closing comment around, so I had to manually remove them.